lib/util/mime_utils.js

```diff
@@ -107,13 +107,13 @@ shaka.util.MimeUtils = class {
     switch (shaka.util.MimeUtils.getCodecBase(codecString)) {
       case 'avc1':
       case 'avc3':
-        return 'avc';
+        return 'avc';  // H264
       case 'hvc1':
       case 'hev1':
-        return 'hevc';
+        return 'hevc'; // H265
       case 'dvh1':
       case 'dvhe':
-        return 'dovi';
+        return 'dovi'; // Dolby Vision
     }
     return codecString;
```

Comment:
Ah, I overlooked this before.  For codecs that aren't in the map, we should return the base, not the full codec string.  Can you put that base codec in a local variable, switch on that, and then return the base here instead of the full thing?